lib/active_record_host_pool/pool_proxy.rb

```diff
@@ -129,7 +129,16 @@ def _connection_pools
     end
 
     def _pool_key
-      @_pool_key ||= "#{@config[:host]}/#{@config[:port]}/#{@config[:socket]}/#{@config[:username]}/#{@config[:slave] && 'slave'}"
+      @_pool_key ||= begin
+        [
+          "#{@config[:host]}
```

Comment:
syntax looks wrong ... should be `@config[:host],\n...`

Reply:
Ah, you're right. Updated and re-pushed in 5db45df0f0711489b111412117ed6ebb3026c110